tools/bcl-test-importer/Makefile

```diff
@@ -0,0 +1,29 @@
+DESTDIR ?= /usr/local/bin/
+CONFIGURATION ?= Debug
+BCL_TEST_IMPORTER_SRC:=BCLTestImporter/ApplicationOptions.cs \
+					BCLTestImporter/BCLTestImporter.csproj \
+					BCLTestImporter/ConsoleWriter.cs \
+					BCLTestImporter/Program.cs \
+					BCLTestImporter/BCLTestProjectGenerator.cs \
+					BCLTestImporter/RegisterTypeGenerator.cs \
+					BCLTestImporter/BCLTestAssemblyDefinition.cs \
+					BCLTestImporter/BCLTestProjectDefinition.cs \
+
+all: BCLTestImporter.exe
+
+BCLTestImporter.exe: $(BCL_TEST_IMPORTER_SRC) Makefile 
+	nuget restore BCLTestImporter/BCLTestImporter.sln
+	msbuild /p:Configuration=$(CONFIGURATION) BCLTestImporter/BCLTestImporter.sln
```

Comment:
do we need `$BCL_TEST_IMPORTER_SRC` ? which needs to be manually updated whenever the solution is modified. msbuild can take care of dependencies

Or maybe just use a wildcard of `*.cs` ? that way the update won't be required in most cases